apps/condo/pages/settings/index.tsx

```diff
@@ -139,6 +141,7 @@ const SettingsPage: PageComponentType = () => {
             </Head>
             <PageWrapper>
                 <OrganizationRequired>
+                    {TestCustomTranslation}
```

Comment:
Almost perfect, but I don't think that's necessary to provide an example inside code. IMO it'll be better to add that to docs

Reply:
Yep, that is correct :-) I'll remove that once we have discussed this proposal internally :-)